test/liquid-comment/fixed.liquid

```diff
@@ -6,6 +6,29 @@
 
 {% comment %}theme-check-disable foo{% endcomment %}
 
+It should support nested comments
+{% comment %}
+  ho
+  {% comment %}
+    hi
+  {% endcomment %}
+  ha
+{% endcomment %}
+
+It should support empty comments
+{% comment %}{% endcomment %}
+
+It should support nested empty comments
+{% comment %}{% comment %}{% endcomment %}{% endcomment %}
```

Comment:
(out of scope for this PR) testing locally i see unempty, nested comments formatted like this remain in one line. does it make sense to format something like this into multiple lines with indentation? or is there a reason we wouldnt do this?

example test i wrote and ran:
```
 ########## INPUT
      {% comment %}ho{% comment %}hi{% endcomment %}{% endcomment %}
      ########## ACTUAL
      {% comment %}ho{% comment %}hi{% endcomment %}{% endcomment %}
      ##########
      + expected - actual

      -{% comment %}ho{% comment %}hi{% endcomment %}{% endcomment %}
      +{% comment %}
      +  ho
      +  {% comment %}
      +    hi
      +  {% endcomment %}
      +{% endcomment %}
```

Reply:
My rationale for comments is that sometimes we use them to "illustrate" stuff. e.g. 

https://github.com/charlespwd/worldviewjs/blob/master/src/transform-world.js#L17-L42

As such, I feel like we _don't_ want to be too smart about how we reformat comments. I feel like, usually, the formatting is intentional and shouldn't be changed.